components/post/default.htm

```diff
@@ -17,11 +17,11 @@
 {% endif %}
 
 <p class="info">
-    Posted
-    {% if post.categories.count %} in
+    {{ 'rainlab.blog::lang.post.posted_on_prefix'|trans }}
+    {% if post.categories.count %} {{ 'rainlab.blog::lang.post.posted_on_in_category'|trans}}
         {% for category in post.categories %}
             <a href="{{ category.url }}">{{ category.name }}</a>{% if not loop.last %}, {% endif %}
         {% endfor %}
     {% endif %}
-    on {{ post.published_at|date('M d, Y') }}
+    {{ 'rainlab.blog::lang.post.posted_on_mid'|trans }} {{ post.published_at|date('rainlab.blog::lang.post.posted_on_date_format'|trans) }}
```

Comment:
I would prefer that this was a single translation key where the date was passed in as a translation variable. @bennothommo thoughts on that? Perhaps you could help @andibraeu out if they don't know how to do that.

Reply:
I'm not sure how that should work, as the date format also depends on the translation

Reply:
you can pass a translated value in as a variable, just starts to get a bit nested. I'm thinking we should do it that way since RTL would have the date before the word

Reply:
I don't have a glue how that should work. Maybe I'm not good enough in reading docs or googling the problem.
But to be honest, I also don't want to get a twig expert, I'm just a user of this plugin that wanted to help getting things improved fast.

Reply:
Fair enough. @bennothommo when you have time can you clean it up a bit with my suggestions / any of your own ideas?

Reply:
@LukeTowers I agree. I'll try and sort it out tonight.